map-prototype/nodes.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
@dcwalk Github renders this with OSM, so we can potentially do this with OSM instead, or make gmaps vs. osm a config?